web/client/components/mapcontrols/search/SearchBar.jsx

```diff
@@ -38,7 +38,8 @@ let SearchBar = React.createClass({
         blurResetDelay: React.PropTypes.number,
         typeAhead: React.PropTypes.bool,
         searchText: React.PropTypes.string,
-        style: React.PropTypes.object
+        style: React.PropTypes.object,
+        displaycrs: React.PropTypes.string
```

Comment:
Are you using this in a custom search action? Probably we could just introduce a generic searchOptions object (where you can put crs and other stuff if you need them) to be passed through to the action. What do you think?